Resources/Locale/en-US/_DV/paper/paper-misc.ftl

```diff
@@ -20,6 +20,16 @@ book-text-subscription-notice = [head=2]Greetings loyal subscriber,[/head]
 
       Signing off,
 
+book-text-centcomm-chads-subscription-notice = [head=2]Greetings loyal subscriber,[/head]
+
+      We hope you enjoy this months edition of CentComm Chads™ sponsored by Nanotrasen!
```

Comment:
Missing possessive apostrophe
```suggestion
      We hope you enjoy this month's edition of CentComm Chads™ sponsored by Nanotrasen!
```
Can just as well fix it in the gorlex description 10 lines above too